tests/tom-swifty-test/SwiftReflector/PropertyWrapTests.cs

```diff
@@ -190,6 +190,7 @@ public void ComputedPropertyTestString ()
 		}
 
 		[Test]
+		[Ignore ("apple's top level let issue")]
```

Comment:
Maybe add a link to Apple's issue in the ignore message (or as a comment)?

Reply:
```suggestion
		[Ignore ("Ignore due to  https://bugs.swift.org/browse/SR-13790")]
```

I think that is the correct one